daprdocs/content/en/developing-applications/building-blocks/pubsub/pubsub-bulk.md

```diff
@@ -269,6 +269,127 @@ Invoke-RestMethod -Method Post -ContentType 'application/json' -Uri 'http://loca
 
 {{< /tabs >}}
 
+## Subscribing messages in bulk
+
+The bulk subscribe API allows you to subscribe multiple messages from a topic in a single request. 
+As we know from [How to: Publish & Subscribe to topics]({{< ref howto-publish-subscribe.md >}}), there are two ways to subscribe to topic(s):
+
+- **Declaratively**, where subscriptions are defined in an external file.
+- **Programmatically**, where subscriptions are defined in user code.
+
+To Bulk Subscribe to topic(s), we just need to use `bulkSubscribe` spec attribute, something like following:
+
+```yaml
+apiVersion: dapr.io/v2alpha1
+kind: Subscription
+metadata:
+  name: order-pub-sub
+spec:
+  topic: orders
+  routes: 
+    default: /checkout
+  pubsubname: order-pub-sub
+  bulkSubscribe:
+    enabled: true
+    maxMessagesCount: 100
+    maxAwaitDurationMs: 40
+scopes:
+- orderprocessing
+- checkout
+```
+
+In the example above, `bulkSubscribe` altogether is _optional_. If you decide to use `bulkSubscribe`, then:
+- `enabled` is mandatory
+- You can optionally configure the max number of messages (`maxMessagesCount`) delivered via Dapr in a bulk message
+- You can optionally provide the max duration to wait (`maxAwaitDurationMs`) before a bulk message is sent via Dapr to the app
+
+If the application decides not to configure `maxMessagesCount` and/or `maxAwaitDurationMs`, the defaults will be used as per the respective component in Dapr.
+
+The application receives an EntryId associated with each entry (individual message) in the bulk message. This EntryId must be used by the app to communicate the status of that particular entry. If the app fails to notify on an EntryId status, it's considered a `RETRY`.
+
+Status | Description
```

Comment:
Not clear how this table below relates to EntryId. Can you explain with an EntryId example?

Reply:
I have incorporated more details. Please see if it suffices.